internal/uvm/wait.go

```diff
@@ -1,6 +1,19 @@
 package uvm
 
+import (
+	"github.com/Microsoft/hcsshim/internal/logfields"
+	"github.com/sirupsen/logrus"
+)
+
 // Waits synchronously waits for a utility VM to terminate.
 func (uvm *UtilityVM) Wait() error {
-	return uvm.hcsSystem.Wait()
+	err := uvm.hcsSystem.Wait()
+
+	logrus.WithField(logfields.ContainerID, uvm.ID).
+		Debug("UVM exited, waiting for output processing to complete")
+	if uvm.outputProcessingDone != nil {
```

Comment:
Dont you want these in the other order. Wait for logs, wait for uvm to finish terminating?

Reply:
I'm not sure it matters, but I was thinking there might be more output arriving as the VM terminates, so we'd do things in this order to ensure we don't ignore any output.

Reply:
Ok